config.toml

```
@@ -40,6 +40,11 @@ sphinxopts = [
     '-D latex_elements.fontenc=\\usepackage{fontspec}',
 ]
 
+[languages.hu]
+name = "Hungarian"
+translated_name = "magyar"
```

I asked a Hungarian friend who said that they would capitalise this:

```suggestion
translated_name = "Magyar"
```